users/tests/test_unit_customuser_model.py

```diff
@@ -71,7 +71,7 @@ def test_full_clean_method_should_be_called_on_save(mocked_full_clean):
         user = CustomUser()
         user.save()
 
-        mocked_full_clean.assert_called_once()
+        mocked_full_clean.assert_called()
```

Comment:
How many times now `full_clean` is called that you changed this? :P

Reply:
`CustomUserPreferences` inherits from `BaseModel` (abstract model) now. If I am not wrong, as we added signals to create object of `CustomUserPreferences` it will also call `full_clean()` for that creation.